app/src/main/java/com/google/droidjump/MainActivity.java

```diff
@@ -150,6 +175,29 @@ protected void onResume() {
         }
     }
 
+    public Task<AnnotatedData<PlayerBuffer>> loadFriendNames() {
+        return getPlayersClient().loadFriends(GameConstants.FRIENDS_PER_PAGE, /* forceReload = */ false)
+                .addOnSuccessListener(buffer -> {
+                    playerBuffer = buffer.get();
+                    try {
```

Comment:
What can fail there and why not catching an exception that is thrown?

Reply:
Fixed. That how it was coded on the project from play game services android samples on Github.